third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -5283,8 +5283,8 @@ void OnNewWindow1(v8::Isolate* isolate, LocalFrame* localFrame) {
                        TestEnv("RECORD_REPLAY_DISABLE_SOURCEMAP_CACHE")));
 
 
-  DefineProperty(isolate, args, "CDPERROR_MISSINGCONTEXT", 
-                 v8::Integer::New(isolate, CDPERROR_MISSINGCONTEXT));
+  DefineProperty(isolate, args, "CDPERROR_MISSINGCONTEXT",
```

Comment:
Why can't it do `Integer`?

Reply:
I think this was causing a replay error--there's no such thing as an integer in JS, just 'number'.

Reply:
I guess I meant the cast to `double` (not sure, why this comment is tagged to the wrong line 😆)